exec/csnex/Exec.cs

```diff
@@ -908,10 +945,41 @@ void RESETC()
         }
 #endregion
 #endregion
+        void dump_frames()
+        {
+            #if DUMP_FRAMES
+            Console.Out.Write("Frames:\n");
+            for (int i = 0; i < frames.Count; i++) {
+                ActivationFrame f = frames[i];
+                Console.Out.Write("\t {04:0}\t{ locals={04:1} }\n", i, f.FrameSize);
+                for (int l = 0; 0 < f.FrameSize; l++) {
```

Comment:
You seem to have removed the `FrameSize` member but did not update this code. (This is why conditional compilation is annoying, because the compiler can't check your code.)

Reply:
Opps.  Yea, I don't care too much for the conditional complication either, but that's about the best option in C#.  I fixed this up so it is "current".

Reply:
Conditional compilation is not the answer here. Use a constant (static) boolean value, and do `if (DUMP_FRAMES) ...`. Then when `DUMP_FRAMES` is `false`, the compiler will compile the code inside but won't emit any instructions for it. That is a far better option than `#if`.